servers.json

```diff
@@ -6,6 +6,13 @@
             "edenmine.eu"
         ]
     },
+    {
+        "id": "ukiyo",
```

Comment:
Is `ukiyo.rip` and `ukiyomc.com` (this pr) the same server?

If they are, which one would you like to keep, as the entries are conflicting:

![image](https://user-images.githubusercontent.com/62861393/142922032-745f3d7b-4f58-4838-9c94-ecfabc661591.png)

Reply:
Different servers.

Reply:
Hey there @GrowlyX, these are different servers

Reply:
> Hey there @GrowlyX, these are different servers

Is it okay if I change the id of your server to `ukiyomc`, so there are no conflicts with the other server?

Reply:
Sure, though when I created the pull requests I saw no upstream conflicts